sql/src/test/java/org/apache/druid/sql/calcite/util/CalciteTests.java

```diff
@@ -300,7 +300,7 @@ public AuthenticationResult createEscalatedAuthenticationResult()
           new TimestampSpec(TIMESTAMP_COLUMN, "iso", null),
           new DimensionsSpec(
               ImmutableList.<DimensionSchema>builder()
-                  .addAll(DimensionsSpec.getDefaultSchemas(ImmutableList.of("dim1", "dim2", "dim3", "dim4", "dim5")))
+                  .addAll(DimensionsSpec.getDefaultSchemas(ImmutableList.of("dim1", "dim2", "dim3", "dim4", "dim5", "dim6")))
```

Comment:
Please confirm once if other tests are not failing on changing the row shape.

Reply:
Confirmed. No other tests are failing. Travis passes too